tests/test_redirect_uri_config.py

```diff
@@ -31,6 +31,7 @@ def test_redirect_uri_config(self):
         redirect_uri_config = RedirectUriConfig.from_config(config)
         assert redirect_uri_config.full_uri == 'https://myexample.com:6000/callback'
         assert redirect_uri_config.endpoint == 'callback'
+        assert repr(redirect_uri_config) == f'({redirect_uri_config.full_uri}, {redirect_uri_config.endpoint})'
```

Comment:
Unrelated to this PR and I'm not sure it's needed to test the string representation for this class (which isn't intended to be used for anything).

Reply:
To increase the test coverage. It's at 98.86% now.

Reply:
While I appreciate your effort, striving for 100 % coverage is not really necessary. 🙂 
I think it's pushing against the limits of the trade-off of having more test code to maintain.